supertokens_python/recipe/passwordless/emaildelivery/service/smtp/pless_email.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+from os import path
+from string import Template
+from typing import TYPE_CHECKING
+
+from supertokens_python.ingredients.emaildelivery.service.smtp import \
+    GetContentResult
+
+from supertokens_python.supertokens import Supertokens
+
+if TYPE_CHECKING:
+    from supertokens_python.recipe.passwordless.interfaces import TypePasswordlessEmailDeliveryInput
+
+
+def pless_email_content(email_input: TypePasswordlessEmailDeliveryInput) -> GetContentResult:
+    # TODO: FIXME (func name, email title, etc)
+    supertokens = Supertokens.get_instance()
+    app_name = supertokens.app_info.app_name
+    body = get_pless_email_html(app_name, email_input.email, str(email_input.code_life_time))
```

Comment:
need to correctly humanise the code_life_time

Reply:
Mentioned in TODO. File renamed as `pless_login.py`